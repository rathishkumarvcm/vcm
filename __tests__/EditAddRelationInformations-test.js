import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';
import EditAddRelationInformations from '../App/Screens/EditFamilyMemberInformation/EditFamilyMemberComponent';

const navigation = { navigate: jest.fn(), getParam: jest.fn()};

describe('Add Relationship Information', () => {

    // Application Screnshot
    it('snap shot', () => {
        const tree = renderer.create(<EditAddRelationInformations navigation={navigation} />).toJSON();
        expect(tree).toMatchSnapshot();
    });

    test('render',()=>{
        const component = shallow(<EditAddRelationInformations navigation={navigation} />);  
        component.render();
      });
});