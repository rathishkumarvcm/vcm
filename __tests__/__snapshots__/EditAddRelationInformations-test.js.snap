// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Relationship Information snap shot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#F7FAFF",
      "flex": 1,
      "width": "100%",
    }
  }
>
  <View
    accessible={true}
    clickable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "flex": 0.12,
        "flexDirection": "row",
        "opacity": 1,
        "paddingLeft": "4%",
        "paddingRight": "4%",
        "width": "100%",
      }
    }
  >
    <View
      accessible={true}
      clickable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "height": 100.5,
          "opacity": 1,
          "width": "25%",
        }
      }
    >
      <Image
        resizeMode="contain"
        source={
          Object {
            "testUri": "../../../App/Images/logo.png",
          }
        }
        style={
          Object {
            "alignItems": "flex-start",
            "justifyContent": "flex-start",
            "width": "100%",
          }
        }
      />
    </View>
    <View
      style={
        Object {
          "width": "12%",
        }
      }
    />
    <View
      accessible={true}
      clickable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "marginTop": 24.5,
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "height": 40,
            "width": 56,
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "black",
                "fontSize": 40,
              },
              Object {
                "backgroundColor": "transparent",
                "marginHorizontal": 5,
              },
              Object {
                "fontFamily": "EvilIcons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
          testID="iconIcon"
        >
          
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "width": "10%",
        }
      }
    />
    <View
      accessible={true}
      clickable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "height": 30,
            "width": 44,
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "black",
                "fontSize": 30,
              },
              Object {
                "backgroundColor": "transparent",
                "marginHorizontal": 5,
              },
              Object {
                "fontFamily": "Ionicons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
          testID="iconIcon"
        >
          
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      clickable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "height": 30,
            "width": 44,
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "black",
                "fontSize": 30,
              },
              Object {
                "backgroundColor": "transparent",
                "marginHorizontal": 5,
              },
              Object {
                "fontFamily": "Ionicons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
          testID="iconIcon"
        >
          
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      clickable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "flex-end",
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <RNGestureHandlerButton
        collapsable={false}
        onGestureEvent={[Function]}
        onGestureHandlerEvent={[Function]}
        onGestureHandlerStateChange={[Function]}
        onHandlerStateChange={[Function]}
        rippleColor={0}
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            undefined,
          ]
        }
      >
        <View
          accessible={true}
          style={
            Object {
              "alignItems": "flex-end",
              "justifyContent": "center",
              "marginEnd": 10,
              "opacity": 1,
            }
          }
        >
          <View
            style={
              Object {
                "height": 40,
                "width": 56,
              }
            }
          >
            <Text
              allowFontScaling={false}
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 40,
                  },
                  Object {
                    "backgroundColor": "transparent",
                    "marginHorizontal": 5,
                  },
                  Object {
                    "fontFamily": "Material Icons",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {},
                ]
              }
              testID="iconIcon"
            >
              
            </Text>
          </View>
        </View>
      </RNGestureHandlerButton>
    </View>
  </View>
  <RCTScrollView
    style={
      Object {
        "flex": 0.85,
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "flexDirection": "row",
            "marginTop": 29.5,
            "paddingLeft": "4%",
            "paddingRight": "4%",
            "width": "100%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#0000FF",
              "fontSize": 23,
            }
          }
        >
          Pro..
        </Text>
        <Text
          style={
            Object {
              "color": "#56565A",
              "fontSize": 23,
            }
          }
        >
            &gt;  
        </Text>
        <Text
          style={
            Object {
              "color": "#0000FF",
              "fontSize": 23,
            }
          }
        >
          Bas..
        </Text>
        <Text
          style={
            Object {
              "color": "#56565A",
              "fontSize": 23,
            }
          }
        >
            &gt;  
        </Text>
        <Text
          style={
            Object {
              "color": "#56565A",
              "fontSize": 23,
              "fontWeight": "bold",
            }
          }
        >
          Relationship Information
        </Text>
      </View>
      <View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "center",
              "marginTop": 29.5,
              "paddingLeft": "4%",
              "paddingRight": "4%",
              "width": "100%",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#56565A",
                "fontSize": 29.5,
                "fontWeight": "bold",
                "width": "70%",
              }
            }
          >
            Relationship Information
          </Text>
          <Text
            onPress={[Function]}
            style={
              Object {
                "color": "#0000FF",
                "fontSize": 23,
                "textAlign": "right",
                "width": "30%",
              }
            }
          >
            Manage
          </Text>
        </View>
        <View
          style={
            Object {
              "borderBottomWidth": 1,
              "borderColor": "#B2B2B2",
              "marginLeft": "4%",
              "marginRight": "4%",
              "marginTop": 16.5,
            }
          }
        />
        <View>
          <View
            style={
              Object {
                "backgroundColor": "#FFFFFF",
                "borderColor": "#D4D4D4",
                "borderWidth": 1,
                "marginLeft": "4%",
                "marginRight": "4%",
                "marginTop": 33,
                "width": "92%",
              }
            }
          >
            <View
              style={
                Object {
                  "marginTop": 29.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                    "fontWeight": "bold",
                  }
                }
              >
                Name
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 3.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#0000FF",
                    "fontSize": 26.5,
                    "marginBottom": "2%",
                    "marginTop": 5,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "marginTop": 29.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                    "fontWeight": "bold",
                  }
                }
              >
                Prefix
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 3.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#56565A",
                    "fontSize": 26.5,
                    "marginBottom": "2%",
                    "marginTop": 5,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "marginTop": 29.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                    "fontWeight": "bold",
                  }
                }
              >
                Suffix
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 3.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#56565A",
                    "fontSize": 26.5,
                    "marginBottom": "2%",
                    "marginTop": 5,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "marginTop": 29.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                    "fontWeight": "bold",
                  }
                }
              >
                Relationship
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 3.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#56565A",
                    "fontSize": 26.5,
                    "marginBottom": "2%",
                    "marginTop": 5,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "marginTop": 29.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                    "fontWeight": "bold",
                  }
                }
              >
                Social Security Number
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 3.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#56565A",
                    "fontSize": 26.5,
                    "marginBottom": "2%",
                    "marginTop": 5,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "marginTop": 29.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                    "fontWeight": "bold",
                  }
                }
              >
                Date of birth
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 3.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#56565A",
                    "fontSize": 26.5,
                    "marginBottom": "2%",
                    "marginTop": 5,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "marginTop": 29.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                    "fontWeight": "bold",
                  }
                }
              >
                Gender
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 3.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#56565A",
                    "fontSize": 26.5,
                    "marginBottom": "2%",
                    "marginTop": 5,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "marginTop": 29.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                    "fontWeight": "bold",
                  }
                }
              >
                Marital Status
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 3.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#56565A",
                    "fontSize": 26.5,
                    "marginBottom": "2%",
                    "marginTop": 5,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "marginTop": 29.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                    "fontWeight": "bold",
                  }
                }
              >
                Citizenship
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 3.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#56565A",
                    "fontSize": 26.5,
                    "marginBottom": "2%",
                    "marginTop": 5,
                  }
                }
              />
            </View>
          </View>
        </View>
        <View>
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
                "marginTop": 29.5,
                "paddingLeft": "4%",
                "paddingRight": "4%",
                "width": "100%",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#56565A",
                  "fontSize": 29.5,
                  "fontWeight": "bold",
                  "width": "70%",
                }
              }
            >
              Contact Information
            </Text>
            <Text
              onPress={[Function]}
              style={
                Object {
                  "color": "#0000FF",
                  "fontSize": 23,
                  "textAlign": "right",
                  "width": "30%",
                }
              }
            >
              Manage
            </Text>
          </View>
          <View
            style={
              Object {
                "borderBottomWidth": 1,
                "borderColor": "#B2B2B2",
                "marginLeft": "4%",
                "marginRight": "4%",
                "marginTop": 16.5,
              }
            }
          />
          <View
            style={
              Object {
                "backgroundColor": "#FFFFFF",
                "borderColor": "#D4D4D4",
                "borderWidth": 1,
                "height": 230,
                "marginLeft": "4%",
                "marginRight": "4%",
                "marginTop": 33,
                "width": "92%",
              }
            }
          >
            <View
              style={
                Object {
                  "marginTop": 29.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                    "fontWeight": "bold",
                  }
                }
              >
                Mailing Address
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 3.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#56565A",
                    "fontSize": 26.5,
                    "marginBottom": "2%",
                    "marginTop": 5,
                  }
                }
              >
                5400 N Black Oak Lake Rd
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 29.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                    "fontWeight": "bold",
                  }
                }
              >
                Physical Address
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 3.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#56565A",
                    "fontSize": 26.5,
                    "marginBottom": "2%",
                    "marginTop": 5,
                  }
                }
              >
                Same as mailing
              </Text>
            </View>
          </View>
        </View>
        <View>
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
                "marginTop": 29.5,
                "paddingLeft": "4%",
                "paddingRight": "4%",
                "width": "100%",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#56565A",
                  "fontSize": 29.5,
                  "fontWeight": "bold",
                  "width": "70%",
                }
              }
            >
              Phone Information
            </Text>
            <Text
              onPress={[Function]}
              style={
                Object {
                  "color": "#0000FF",
                  "fontSize": 23,
                  "textAlign": "right",
                  "width": "30%",
                }
              }
            >
              Manage
            </Text>
          </View>
          <View
            style={
              Object {
                "borderBottomWidth": 1,
                "borderColor": "#B2B2B2",
                "marginLeft": "4%",
                "marginRight": "4%",
                "marginTop": 16.5,
              }
            }
          />
          <View
            style={
              Object {
                "backgroundColor": "#FFFFFF",
                "borderColor": "#D4D4D4",
                "borderWidth": 1,
                "height": 148,
                "marginLeft": "4%",
                "marginRight": "4%",
                "marginTop": 33,
                "width": "92%",
              }
            }
          >
            <View
              style={
                Object {
                  "marginTop": 29.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                    "fontWeight": "bold",
                  }
                }
              >
                Primary Mobile
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 3.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#56565A",
                    "fontSize": 26.5,
                    "marginBottom": "2%",
                    "marginTop": 5,
                  }
                }
              >
                +1(xxx) xxx - 7890
              </Text>
            </View>
          </View>
        </View>
        <View>
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
                "marginTop": 29.5,
                "paddingLeft": "4%",
                "paddingRight": "4%",
                "width": "100%",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#56565A",
                  "fontSize": 29.5,
                  "fontWeight": "bold",
                  "width": "70%",
                }
              }
            >
              Email Information
            </Text>
            <Text
              style={
                Object {
                  "color": "#0000FF",
                  "fontSize": 23,
                  "textAlign": "right",
                  "width": "30%",
                }
              }
            >
              Manage
            </Text>
          </View>
          <View
            style={
              Object {
                "borderBottomWidth": 1,
                "borderColor": "#B2B2B2",
                "marginLeft": "4%",
                "marginRight": "4%",
                "marginTop": 16.5,
              }
            }
          />
          <View
            style={
              Object {
                "backgroundColor": "#FFFFFF",
                "borderColor": "#D4D4D4",
                "borderWidth": 1,
                "height": 148,
                "marginLeft": "4%",
                "marginRight": "4%",
                "marginTop": 33,
                "width": "92%",
              }
            }
          >
            <View
              style={
                Object {
                  "marginTop": 29.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#333333DE",
                    "fontSize": 26.5,
                    "fontWeight": "bold",
                  }
                }
              >
                Primary
              </Text>
            </View>
            <View
              style={
                Object {
                  "marginTop": 3.5,
                  "paddingLeft": "4%",
                  "paddingRight": "4%",
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#56565A",
                    "fontSize": 26.5,
                    "marginBottom": "2%",
                    "marginTop": 5,
                  }
                }
              >
                paxxxxx@example.com
              </Text>
            </View>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
            "marginTop": 33,
            "paddingLeft": "4%",
            "paddingRight": "4%",
            "width": "100%",
          }
        }
      >
        <View
          accessible={true}
          clickable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFFFFF",
              "borderColor": "#56565A",
              "borderRadius": 1,
              "borderWidth": 1,
              "height": 82,
              "justifyContent": "center",
              "marginLeft": "4%",
              "marginRight": "4%",
              "marginTop": 19.5,
              "opacity": 1,
              "width": "92%",
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#56565A",
                  "fontSize": 26.5,
                },
              ]
            }
          >
            Back
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "flexDirection": "column",
            "marginTop": 33,
            "paddingLeft": "4%",
            "paddingRight": "4%",
            "width": "100%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#56565A",
              "fontSize": 29.5,
            }
          }
        >
          For security reasons, not all information can be uploaded online. Please call us if you need assistance. Phone Number
        </Text>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "marginTop": 39.5,
            "padding": 24.5,
            "width": "100%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#486D89",
              "fontSize": 29.5,
              "marginBottom": 8,
            }
          }
        >
          Investing involves risk including loss of principal.
        </Text>
        <Text
          style={
            Object {
              "color": "#56565A",
              "fontSize": 26.5,
              "justifyContent": "center",
            }
          }
        >
          Victory Mutual Funds and USAA Mutual Funds are distributed by Victory Capital Advisers, Inc. (VCA). VictoryShares ETFs and VictoryShares USAA ETFs are distributed by Foreside Fund Services, LLC (Foreside). VCA and Foreside are members of FINRA and SIPC. Victory Capital Management Inc. (VCM) is the investment adviser to the Victory Mutual Funds, USAA Mutual Funds, VictoryShares ETFs and VictoryShares USAA ETFs. VCA and VCM are not affiliated with Foreside. USAA is not affiliated with Foreside, VCM, or VCA. USAA and the USAA logos are registered trademarks and the USAA Mutual Funds and USAA Investments logos are trademarks of United Services Automobile Association and are being used by Victory Capital and its affiliates under license. Victory Capital means Victory Capital Management Inc., the investment manager of the USAA 529 College Savings Plan (Plan). The Plan is distributed by Victory Capital Advisers, Inc., a broker dealer registered with FINRA and an affiliate of Victory Capital. Victory Capital and its affiliates are not affiliated with United Services Automobile Association or its affiliates. USAA and the USAA logo are registered trademarks and the USAA 529 College Savings Plan logo is a trademark of United Services Automobile Association and are being used by Victory Capital and its affiliates under license.
        </Text>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "white",
            "flexDirection": "column",
            "height": 164.5,
            "paddingBottom": "5%",
            "paddingLeft": "5%",
          }
        }
      >
        <Image
          source={
            Object {
              "testUri": "../../../App/Images/logo.png",
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "white",
            "flexDirection": "row",
            "height": 82,
            "justifyContent": "center",
            "width": "100%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#5D83AE",
              "fontSize": 26.5,
              "fontWeight": "bold",
              "paddingLeft": "4%",
              "width": "50%",
            }
          }
        >
          Privacy Policy
        </Text>
        <Text
          style={
            Object {
              "color": "#5D83AE",
              "fontSize": 26.5,
              "fontWeight": "bold",
              "paddingLeft": "4%",
              "width": "50%",
            }
          }
        >
          Fund Documents
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "white",
            "flexDirection": "row",
            "height": 82,
            "justifyContent": "center",
            "width": "100%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#5D83AE",
              "fontSize": 26.5,
              "fontWeight": "bold",
              "paddingLeft": "4%",
              "width": "50%",
            }
          }
        >
          User Agreement
        </Text>
        <Text
          style={
            Object {
              "color": "#5D83AE",
              "fontSize": 26.5,
              "fontWeight": "bold",
              "paddingLeft": "4%",
              "width": "50%",
            }
          }
        >
          Support
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#56565A",
            "height": 82,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#FFFFFF",
            }
          }
        >
          Copyright Victory Capital Management Inc. ©2020
        </Text>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
